solutions/2.js

```diff
@@ -0,0 +1,16 @@
+const max3 = (num1, num2, num3) => {
+  if (num1 > num2 && num1 > num3){
+    max = num1;
```

Comment:
You can just `return num1` instead of assigning the value to a variable and then return it.